comments/views.py

```diff
@@ -16,7 +16,7 @@
 from django.utils.translation import ugettext as _
 
 from comments.forms import AdminCommentForm, CommentFilterForm
-from comments.models import CannedResponse, CommentModeration
+from comments.models import CannedResponse, CommunityCommentModeration
 
 
 def update(request, comment_pk, action):
```

Comment:
This entrypoint needs to be better protected. I was able to anonymously approve and reject comments by crafting an appropriate URL.